### ISSUE TITLE
[Bug] Can't delete REST API apps

### ISSUE THREAD

**Issue by fredroo** [paypal/PayPal-REST-API-issues#157]
click in delete app, put app name and show for me this error

![image](https://user-images.githubusercontent.com/6863089/31633734-b116cdf0-b297-11e7-8403-aeacdbc3d538.png)

Brazilian Account

email: financeiro@livetechtecnologia.com.br

**Comment by fredroo** [comment]
@braebot can help me with this?

**Comment by fredroo** [comment]
@pp-randy  can help me with this?

**Comment by fredroo** [comment]
@xiaoleih41 can help me with this?

**Comment by pp-randy** [comment]
@fredroo have you already engaged [PayPal Technical Support](https://www.paypal-techsupport.com/) for this? They should be able to help you resolve this specific issue.  I tested REST App Deletion and it was working so most likely it will require the technical support team to resolve.

**Comment by permutations** [comment]
This is a real bug. I encountered it, too, and I think I can tell you how to reproduce it.

At the bottom of this page:

https://developer.paypal.com/developer/applications/

...there was a button that invited me to create Webhooks for my NVP/SOAP app. I had already created a Webhook for a REST app. This cross-linked something. I now have a Webhooks app that is listed under the REST apps but cannot be deleted and doesn't seems to be connected to anything.

I can delete REST apps that are created in the usual way, but not this one that was originally supposed to be a Webhook for NVP/SOAP, but somehow ended up listed as a REST app.
